packages/sanity/src/core/form/inputs/ReferenceInput/ReferencePreview.tsx

```diff
@@ -104,13 +110,11 @@ export function ReferencePreview(props: {
                 content={
                   <Box padding={2}>
                     <Text size={1}>
-                      {preview.draft?._updatedAt ? (
-                        <>
-                          Edited <TimeAgo time={preview.draft._updatedAt} />
-                        </>
-                      ) : (
-                        <>No unpublished edits</>
-                      )}
+                      <Text size={1}>
+                        {preview.draft?._updatedAt
+                          ? `Edited ${timeSinceEdited}`
+                          : 'No unpublished edits'}
+                      </Text>
```

Comment:
```suggestion
  {preview.draft?._updatedAt
    ? `Edited ${timeSinceEdited}`
    : 'No unpublished edits'}
```

No need to double wrap `<Text>`, as doing so messages with padding here:

<img width="130" alt="image" src="https://github.com/sanity-io/sanity/assets/209129/ac2b67b5-2017-4677-8979-2c6dbc941112">